code/modules/client/preferences.dm

```diff
@@ -149,23 +149,31 @@ datum/preferences
 		save_character()
 
 /datum/preferences/proc/getMinAge(var/age_min)
-	if(species == "Vaurca Worker" || species == "Vaurca Warrior" || species == "Machine" || species == "Diona")
+
+	var/datum/species/mob_species = all_species[species]

```

Comment:
Are two procs necessary? Find a way to remove one. If preferences.getMinAge() is used somewhere, then keep it. The lower class items can access it with pref.getMinAge().